scripts/utils.ps1

```diff
@@ -46,14 +46,23 @@ function Write-Header([string]$text) {
 function Get-ExecutablePath([string]$name, [string]$directory, [string]$envVar) {
     $path = [environment]::GetEnvironmentVariable($envVar, "Process")
 
-    if (!$path) {
-        if (!$directory) {
-            $path = Exec { & where.exe $name } | Select-Object -First 1
-        }
-        else {
-            $path = Exec { & where.exe /R $directory $name } | Select-Object -First 1
+    try
+    {
+        if (!$path) {
+            if (!$directory) {
+                $path = Exec { & where.exe $name } | Select-Object -First 1
+            }
+            else {
+                $path = Exec { & where.exe /R $directory $name } | Select-Object -First 1
+            }
         }
     }
+    catch
+    {
+        Write-Error "Failed to locate executable '${name}' on the path"
```

Comment:
I would recommend to throw from here instead of `Write-Error` + `exit 1` so we can let the calling script decide what to do (by default it is going to be write + exit).

Reply:
I used _exit_ because there was already one in the method. I'll change both to _throw_.